docs/en/Overview.md

```diff
@@ -34,87 +34,107 @@ with the largest deployment exceeding 1,500 nodes.
+* The worker is the most important component, as it stores both metadata and data that are sharded by key, usually the path of the file.
+* The client runs inside the applications and utilizes the same consistent hash algorithm to determine the appropriate worker for the corresponding file.
+* The service registry is responsible for service discovery and maintains a list of workers.
+* The scheduler handles all asynchronous jobs, such as preloading data to workers.
 
-1. **Intelligent Multi-tiering Caching**: Alluxio clusters act as a read and write cache for data in
-connected storage systems. Configurable policies automatically optimize data placement for
-performance and reliability across both memory and disk (SSD/HDD). Caching is transparent to the
-user and uses buffering to maintain consistency with persistent storage. See
-[Alluxio Storage Management]({{ '/en/core-services/Caching.html' | relativize_url }}) for more details.
+## Technical Highlights
 
-1. **Server-Side API Translation**: Alluxio supports industry common APIs, such as HDFS API, S3 API,
-FUSE API, REST API. It transparently converts from a standard client-side interface to any storage
-interface. Alluxio manages communication between applications and file or object storage,
-eliminating the need for complex system configuration and management. File data can look like object
-data and vice versa.
+### Caching Data Affinity
 
-To understand more details on Alluxio internals, please read
-[Alluxio architecture and data flow]({{ '/en/overview/Architecture.html' | relativize_url }}).
+The client obtains a list of DORA workers from a highly available service registry to support tens of thousands of Alluxio workers.
+The client uses a consistent hashing algorithm to determine which worker to visit based on the file path as the key,
+ensuring that the same file always goes to the same worker for a maximum cache hit rate.
+This avoids a performance bottleneck because a client will directly interface with the appropriate worker
+without needing to refer to the service registry,
 
-## Getting Started
+In addition, DORA's architecture allows for easy scalability by adding more nodes to the cluster.
+Each worker node can support tens of millions of files, making it easy to handle increasing data volumes and growing user bases.
 
-To quickly get Alluxio up and running, take a look at our
-[Getting Started]({{ '/en/overview/Getting-Started.html' | relativize_url }}) page,
-which explains how to deploy Alluxio and run examples in a local environment.
+### Paging Data Store
 
-Also try our getting started tutorial for Presto & Alluxio via:
+DORA uses a paging store model as its cache storage, offering finer-grained caching for small to medium-sized read requests on large files.
+DORA's fine-grained caching has resulted in up to 150x read amplification and improved unstructured file position read by up to 9x.
```

Comment:
performance comparisons should explain the reference. 150x from what? prefer to remove them as they don't offer the reader any useful info. should explain what the concept is and why it is a good choice, not spit out baseless comparisons

Reply:
lol those are copied from blog....